CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+# Contributing to ubuntu-image
+
+This document covers the following topics:
+
+* [Setting up](#setting-up)
+* [Building](#building)
+* [Testing](#testing)
+* [Release process](#release-process)
+
+
+## Setting up
+
+Ubuntu 18.04 LTS or later is recommended for ubuntu-image development.
+Usually, the [latest LTS](https://releases.ubuntu.com/) would be the best choice.
+
+Go 1.21 [(or later)](https://go.dev/dl/) is required to build ubuntu-image.
+
+
+## Building
+
+1. Clone this repository in a directory where you have read-write permissions, such as your home directory: 
+```
+cd ~/
+git clone https://github.com/canonical/ubuntu-image.git`
+```
+2. Change into the newly cloned repository:
+```
+cd ubuntu-image
+```
+3. Run:
+```
+go build -o . ./...
```

Comment:
```suggestion
CGO_ENABLED=0 go build -o . ./...
```

This is closer to how we build the binary in the snap and may avoid differences in behavior with the snap.